### ISSUE TITLE
Crash when selecting non-default trackers

### ISSUE THREAD

**Issue by Alvin21Bon** [veger/TLBE#62]
Platform: Linux
Factorio version: 2.0.8
TLBE version: 2.0.0

When adding the trackers, **Area Tracker & City Block Tracker**, the following error message appears and crashes the save:

```
The mod Time Lapse Base Edition (2.0.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event TLBE::on_gui_selection_state_changed (ID 62)
Unknown sprite "utility/show_player_names_in_map_view_black"
stack traceback:
	[C]: in function 'add'
	__TLBE__/scripts/gui.lua:1502: in function 'createTrackerConfigAndInfo'
	__TLBE__/scripts/gui.lua:469: in function <__TLBE__/scripts/gui.lua:417>
```

**Comment by veger** [comment]
Thanks for reporting! I thought I fixed all changed/updated prototypes for the GUI, but this one slipped through.

It is fixed in v2.0.1

**Comment by Alvin21Bon** [comment]
Great! Thanks for this mod
